spec/schemas/security._common.yaml

```diff
@@ -321,27 +389,45 @@ components:
     DynamicOptions:
       type: object
       properties:
-        filteredAliasMode:
+        filtered_alias_mode:
           type: string
-        disableRestAuth:
+        disable_rest_auth:
           type: boolean
-        disableIntertransportAuth:
+        disable_intertransport_auth:
           type: boolean
-        respectRequestIndicesOptions:
+        respect_request_indices_options:
           type: boolean
-        kibana: {}
-        http: {}
-        authc: {}
-        authz: {}
-        authFailureListeners: {}
-        doNotFailOnForbidden:
+        opensearch-dashboards:
+          type: object
+        kibana:
+          type: object
+        http:
+          type: object
+        authc:
+          type: object
+        authz:
+          type: object
+        auth_failure_listeners:
+          type: object
+        do_not_fail_on_forbidden:
           type: boolean
-        multiRolespanEnabled:
+        multi_rolespan_enabled:
           type: boolean
-        hostsResolverMode:
+        hosts_resolver_mode:
           type: string
-        doNotFailOnForbiddenEmpty:
+        do_not_fail_on_forbidden_empty:
           type: boolean
+        on_behalf_of:
+          type: object
+
+    # Forbidden:
```

Comment:
Put back.

Reply:
Do you mean remove this comment or uncomment it?

I've kept these commented line so that its easier to address payload schema validation for response other than 200 by uncommenting them, once support is added. I understand its dead-code but thought it might be easier for some-one who picks this up next.

Reply:
I understand now. From the comment alone it was not clear why this code is commented. I am good with a TODO above it that explains it.